desktop/src/main/java/bisq/desktop/primary/main/content/wallet/send/WalletSendModel.java

```diff
@@ -24,6 +24,7 @@
 public class WalletSendModel implements Model {
     private final StringProperty addressProperty = new SimpleStringProperty(this, "address");
     private final StringProperty amountProperty = new SimpleStringProperty(this, "amount");
+    private final StringProperty passphraseProperty = new SimpleStringProperty(this, "passphrase");
 
     public StringProperty addressProperty() {
```

Comment:
I don't follow the JavaFx convention of offering properties and the protpery value in getters as it causes more boilerplate code and Lombok does not support those types of getters unfortunately. But this version is for sure more the JavaFx standard way...